group_vars/ethiopia_demo/vault.yml

```diff
@@ -0,0 +1,80 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
Let's alphabetise these while we're at it.

Reply:
Oops, lost sight of that halfway through, thanks.